core-common/src/main/java/eu/arrowhead/common/database/entity/ChoreographerStep.java

```diff
@@ -90,6 +84,10 @@
     @OnDelete (action = OnDeleteAction.CASCADE)
     private Set<ChoreographerRunningStep> runningSteps = new HashSet<>();
 
```

Comment:
should move step details here (in the new approach, there is only one step details per step)